### PR TITLE
[open-in-visual-studio-code] Also work with VSCodium

### PR DESCRIPTION
**Pull request by chohner**
## Description
Enables [VSCodium](https://github.com/VSCodium/vscodium) to open the current Finder folder

## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/open-in-visual-studio-code/src/index.ts [hunk 1]
```diff
@@ -32,7 +32,9 @@ const getSelectedFinderWindow = (): Promise<string> => {
 
 export default async () => {
   const applications = await getApplications();
-  const visualStudioCode = applications.find((app) => app.bundleId === "com.microsoft.VSCode");
+  const visualStudioCode = applications.find(
+    (app) => app.bundleId === "com.microsoft.VSCode" || app.bundleId === "com.vscodium"
+  );
```
Comment (thomaspaulmann) [inline extensions/open-in-visual-studio-code/src/index.ts:37]: This logic can have different results, depending on the order of the applications. Also, what happens to people that have both versions installed? 

I would suggest to do the following:
1. Search for VS Code and VS Codium separately
2. Combine the results with an optional ternary operator. 

Alternatively, you can order the applications by bundle identifier and then use `find` as you do. An improvement could be to add a preference to the extension as a dropdown to select which version of VS Code should be used. By default, you could have an "automatic" way that prefers VS Code over Codium.

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Thanks for the addition, this is great. I left a comment to improve the logic for detecting VSCodium.

**Comment by chohner** [comment]
The last commit adds a dropdown menu to the preferences (which defaults to `VSCode`)

Screenshot 2022-12-05 at 18 38 35

Does this match what you expect @thomaspaulmann ?

**Comment by thomaspaulmann** [comment]
Looking good! Thanks for the contribution 👏

**Pull request merged**